plugins/scaffolder/src/components/fields/MultiEntityPicker/MultiEntityPicker.tsx

```diff
@@ -64,40 +66,35 @@ export const MultiEntityPicker = (props: MultiEntityPickerProps) => {
     uiSchema['ui:options']?.defaultNamespace || undefined;
 
   const catalogApi = useApi(catalogApiRef);
-
+  const entityPresentationApi = useApi(entityPresentationApiRef);
   const { value: entities, loading } = useAsync(async () => {
     const { items } = await catalogApi.getEntities(
       catalogFilter ? { filter: catalogFilter } : undefined,
     );
-    return items;
+    const primaryTitles: string[] = [];
```

Comment:
I think this change shouldn't be needed as you are now using `EntityDisplayName` for rendering the options which will invoke the entity presentation api internally 🤔

Reply:
@vinzscam primaryTitles are added in `useAsync` to remove `getLabel` which is used in value of `Autocomplete` as it needs primaryTitle of the entities. Though if we want to completely remove `value` field of the `Autocomplete` adding primaryTitle in useAsync is no longer need. I did changes based on the requested changes in another similar PR: https://github.com/backstage/backstage/pull/24081#discussion_r1557386266